components/centraldashboard/public/components/main-page.css

```diff
@@ -19,11 +19,6 @@
     flex: 1
 }
 
-.bottom {
-    margin-top: auto;
```

Comment:
This line is needed for bottom aligning the tabs, which is how the tabs tend to behave

Reply:
I think it was needed in the case when it was part of the top-navigation bar because it's overall height was larger than the standard height of the tabs (48px). Once we move it to its own section, it shouldn't matter since the height of the parent container will be determined by the height of the paper-tabs.

Before:
![ePQXQKwvYXL](https://user-images.githubusercontent.com/6835846/57623435-c04a1680-755d-11e9-890c-9dbfa4b5708f.png)


After:
![uaGAcjbS5k4](https://user-images.githubusercontent.com/6835846/57623429-bc1df900-755d-11e9-9204-4c3e4ad202a3.png)

Reply:
I see, thanks!